tests/unit/multi-currency/notes/test-class-note-multi-currency-available-test.php

```diff
@@ -36,7 +36,15 @@ public function test_get_note() {
 		$this->assertSame( 'wc-payments-notes-multi-currency-available', $actions->name );
 		$this->assertSame( 'Set up now', $actions->label );
 		$this->assertStringStartsWith( 'admin.php?page=wc-admin&path=/payments/multi-currency-setup', $actions->query );
-		$this->assertSame( true, $actions->primary );
+
+		/**
+		 * The $primary property was deprecated from WooCommerce core. Keeping this to maintain the compatibility with old WooCommerce versions.
+		 * @see https://github.com/woocommerce/woocommerce/blob/ff2d7d704a8f72aeb4990811b6972097aa167bea/plugins/woocommerce/src/Admin/Notes/Note.php#L623-L623.
+		 * @see https://github.com/woocommerce/woocommerce-admin/pull/8474
+		 */
+		if ( isset( $actions->primary ) ) {
```

Comment:
This is to fix error `Undefined property: stdClass::$primary`. 

I double checked and confirmed that our code base only uses this `primary` property in one place and it looks good.

https://github.com/Automattic/woocommerce-payments/blob/e17ed0ebb8b4fa7242e21baf6db1be6c7a0ef970/includes/notes/class-wc-payments-remote-note-service.php#L99